spec/services/change_offer_spec.rb

```diff
@@ -7,33 +7,38 @@
     ChangeOffer.new(actor: provider_user,
                     application_choice: application_choice,
                     course_option: course_option,
-                    conditions: conditions)
+                    conditions: new_conditions)
   end
 
   describe '#save!' do
-    let(:application_choice) { create(:application_choice, :with_offer) }
+    let(:provider_user) do
+      create(:provider_user,
+             :with_make_decisions,
+             providers: [application_choice.offered_option.provider])
+    end
     let(:course_option) { course_option_for_provider(provider: application_choice.course_option.provider) }
-    let(:provider_user) { create(:provider_user, providers: [build(:provider)]) }
-    let(:conditions) { [Faker::Lorem.sentence] }
+    let(:new_conditions) { [Faker::Lorem.sentence] }
+    let(:application_choice) do
+      create(:application_choice, :with_offer, offer: { 'conditions' => ['DBS check'] })
```

Comment:
The `:with_offer` scope already sets up some conditions, so the additional application choice setup is not necessary

https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/factories/application_choice.rb#L155

Reply:
We're overriding the conditions set by the factory to support L41, in order to produce an identical offer.